owasp-top10-2021-apps/a3/copy-n-paste/app/util/db.go

```diff
@@ -126,16 +126,16 @@ func InitDatabase() error {
 
 	dbConn, err := OpenDBConnection()
 	if err != nil {
-		errOpenDBConnection := fmt.Sprintf("OpenDBConnection error: %s", err)
+		errOpenDBConnection := ("OpenDBConnection error: %s" + err)
```

Comment:
Why did you make this modification?

Reply:
It has nothing to do with the vulnerability, I just saw that it is a string formatting practice in Go